core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1047,6 +1051,19 @@ protected function copyChilds($table, $insertID, $id, $parentId)
 
 					$copy[$v][$objCTable->id]['pid'] = $insertID;
 					$copy[$v][$objCTable->id]['tstamp'] = $time;
+
+					try
+					{
+						$this->denyAccessUnlessGranted(
+							ContaoCorePermissions::DC_PREFIX . $this->strTable,
+							new UpdateAction($this->strTable, $objCTable->row(), $copy[$v])
```

Comment:
shouldn't this be `$copy[$v][$objCTable->id]`?

Reply:
Of course :) Fixed in 780b504b9e0a9749e1724c01a5171f73bbb1a7ab.